### PR TITLE
Added business card component

### PR DESCRIPTION
**Pull request by shiladityab24**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [x] 🚩 Other

## Description
Added business card component which will be compatible for all the browsers.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Desktop-View

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
Happy

### CODE REVIEW

**Review by bobbyiliev — CHANGES_REQUESTED** [review]
Hi there! Thank you for this PR! This looks great! Just a couple of suggestions:
- Could you remove the extra white spaces so that the document is formatted well
- I think it would be nice if we set the background color to match the other components, that way it would be consistent with the rest of the components, we could go for `bg-gray-100` for example

Thank you!

File: components/business-card.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                  />
+                  <div class="absolute"></div>
+                </div>
+              </div>
+            </div>
+  
+            <div class="flex flex-col text-right">
+              <button
+                class="flex justify-center max-h-max whitespace-nowrap focus:outline-none focus:ring rounded max-w-max border bg-transparent border-purple-500 hover:border-blue-800 hover:border-indigo-800 flex items-center hover:shadow-lg font-bold py-2 px-4 mr-0 ml-auto"
+              >
```
Comment (bobbyiliev) [inline components/business-card.html]: Also one last thing, maybe move the classes on the same line as the elements throughout the document

Comment (shiladityab24) [reply components/business-card.html]: Understood.

**Comment by shiladityab24** [comment]
Hey @bobbyiliev I have updated the business card component as per your suggested changes.

**Review by bobbyiliev — APPROVED** [review]
Thank you for this contribution!

**Pull request merged**